src/week1/day0/css/style.css

```diff
@@ -0,0 +1,72 @@
+body{
+   font-size: 14px;
+   font-family: serif;
+}
+
+.container1{
+   display: flex;
+   flex-flow: row;
+}
+
+header{
+   background: black;
+   color: white;
+   text-align: center;
+}
+
+nav{
+   width: 25%;
+   background: orange;
```

Comment:
if you are setting only the color use the specific property: background-color. Using background will override every other background property